share/models/validators.py

```diff
@@ -118,7 +118,7 @@ def json_schema_for_field(self, field):
             return rel
         if field.choices:
             return {
-                'enum': field.choices,
+                'enum': [c[0] for c in field.choices],
```

Comment:
I figured we'd want the normalizers to use the key (like `cites`, `is_part_of`, etc.) instead of the display label? Seems less potentially ambiguous.